app/services/application_state_change.rb

```diff
@@ -20,6 +20,11 @@ def initialize(application_choice)
 
     state :awaiting_references do
       event :references_complete, transitions_to: :application_complete
+      event :send_chase_email, transitions_to: :awaiting_references_and_chased
+    end
+
+    state :awaiting_references_and_chased do
```

Comment:
I think the naming of this is passable, what do others think? cc maester @duncanjbrown

Reply:
I think the name is good... but I'm not convinced this is a distinct state on the choice.

This seems like a state of the reference request, not the application choice. And we're opening the door on a world in which `:awaiting_references` is not an honest state because it will miss out some choices which are IRL awaiting references but aren't `:awaiting_references`